class.jetpack.php

```diff
@@ -374,7 +376,7 @@ class Jetpack {
 	/**
 	 * @var Automattic\Jetpack\Connection\Manager
 	 */
-	protected $connection_manager;
+	public $connection_manager;
```

Comment:
Are we good with exposing the connection object already? 😉

Reply:
Why not? Even now you can just do `new Manager ()` and have everything in your hands. I agree with you though that making it public was probably a bit hasty.

Reply:
Since we make things as portable and hookable as possible, I'm fine with that. Just expected to see a method for encapsulating the property access 😉